src/frontend/src/components/navigationLink.ts

```diff
@@ -0,0 +1,15 @@
+import { html, TemplateResult } from "lit-html";
+
+export const navigationLink = (
+  props = {
+    labelText: "About",
+    uuid: "aboutLink",
+    url: "/about",
+  }
+): TemplateResult => html`<a
+  id="${props.uuid}"
+  class="c-nav-links__link t-link t-link--discreet"
+  href="${props.url}"
+  target="_blank"
```

Comment:
is that internal links or external? 

if internal: target blank is correct?
if external: can you add `rel="noopener noreferrer"`?

Reply:
I just left the target as it was. I am not a fan of it, but I get it, since you would loose your state if you wrote something in an input that you did not submit yet.  (Both links are internal ones) cc: @nmattia

Reply:
all good then, thanks for the feedback.